tasks/src/test/java/google/CommonConditions.java

```diff
@@ -1,23 +1,28 @@
-package google_cloud;
+package google;
 
 import org.openqa.selenium.WebDriver;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Listeners;
-import taf.driver.DriverSingleton;
+import taf.browser.BrowserFactory;
+import taf.driver.DriverFactory;
+import taf.listener.SuiteListener;
 import taf.product.google.cloud.model.ComputeEngine;
 import taf.product.google.cloud.service.ComputeEngineFactory;
 import taf.product.google.cloud.service.GoogleCloudService;
 import taf.listener.TestListener;
 
-@Listeners({TestListener.class})
+@Listeners({TestListener.class, SuiteListener.class})
 public class CommonConditions {
   protected WebDriver driver;
 
   @BeforeMethod(alwaysRun = true,
       description = "Google Chrome opens, going to Google Cloud Home page and Calculator page, form is filling")
   public void goToCalculatorPage() {
-    driver = DriverSingleton.getDriver();
+    driver = BrowserFactory.createInstance();
```

Comment:
What is the purpose to initialize WebDriver in the line and reinitialize it in the 24th line?